src/scss/app.scss

```diff
@@ -371,3 +371,14 @@ h4 {
     box-shadow: inset -.5rem -.6rem .6rem 0 rgba(0, 0, 0, .2);
   }
 }
+
+.app__no-border-select {
+  button {
+    border-bottom: 0 !important;
```

Comment:
if you forced to use `!important` rule, always leave a comment on why are doing so

Reply:
ok